### DISCUSSION TITLE
Bug: can't create a notification

### DISCUSSION THREAD

**Discussion by AlexanderKri** [PrefectHQ/prefect#10012] [Q&A]
I suddenly can't create a notification. When I click on "create", the error message appears at the bottom right. 
I have tried turning the server and agent off and on again. I have also deleted the Prefect database. Nothing has helped. Does anyone have any ideas?
![image](https://github.com/PrefectHQ/prefect/assets/55507545/75065e65-99d0-488f-b819-842c7fb9b046)

**Comment by dppss91102** [comment]
I had this issue as well with a freshly installed Prefect server (2.15.0). However, the issue magically went away after I tried to install BitBucket blocks with `pip install prefect-bitbucket`. I have no idea how it solves the issue but wanted to share for anyone also experiencing this.
